discovery-infra/start_discovery.py

```diff
@@ -347,6 +347,10 @@ def nodes_flow(client, cluster_name, cluster, image_path):
             )
             # Validate DNS domains resolvability
             validate_dns(client, cluster.id)
+            if args.wait_for_cvo:
+                cluster_info = client.cluster_get(cluster.id)
+                utils.config_etc_hosts(cluster_info)
```

Comment:
why isnt it the default? Shouldnt we always do some verification on the installed OCP?

Reply:
i think that right now most of the people want to check the usual install. I have added a new target for those that want to wait for CVO and debug.That way we don't interfere with CI and PRs

Reply:
sorry if I was not clear - I specifically addressed  utils.config_etc_hosts(cluster_info), not the whole function

Reply:
aaa-a-a. 
1) right now we need it only for CVO
2) this whole issue should be solved by @YuviGold commit of set_dns. unfortunately, on some servers ( for example HW Lab) this cause issues with resolv.conf, which causes the whole flow to fail (@tsorya wa trying to look into it). I prefer to push it with changes to etc/hosts and not to wait till the set_dns is foixed. Once the fix is in place, i will simply remove etc/hosts configurations